components/dom_struct/lib.rs

```diff
@@ -13,7 +13,7 @@ pub fn dom_struct(args: TokenStream, input: TokenStream) -> TokenStream {
     }
     let attributes = quote! {
         #[derive(deny_public_fields::DenyPublicFields, domobject_derive::DomObject, JSTraceable, MallocSizeOf)]
-        #[crown::unrooted_must_root_lint::must_root]
```

Comment:
This is dangerous, why exactly do we need to cfg_attrs for crown lints, IIRC having lints that are not defined does not cause build to fail (I did non crown builds with manually setting `RUSTC` env to `rustc`)

Reply:
I wanted to get rid of `RUSTC_BOOTSTRAP=script` because it would be nice to be able to just use `cargo build` on a stable toolchain to build the project. `#![feature(register_tool)]` is unstable, and if it is removed then `crown` is not found.

Reply:
If `RUSTC_BOOTSTRAP=script` is present in .cargo/config.toml cargo build should still pick it up, right?

I am not really fan of putting crown lints behind cfg as this PR is already dangerous enough (with not running crown by default), but this can be done sometime in the future.

Reply:
> If RUSTC_BOOTSTRAP=script is present in .cargo/config.toml cargo build should still pick it up, right?

yes, that works. I would have just preferred being able to avoid `RUSTC_BOOTSTRAP` completely for release builds. But I'm fine with dropping the `cfg` parts for now. 

> this PR is already dangerous enough

We could also enable `crown` unconditionally in CI if you would prefer (by checking if the `CI` environment variable is set).

Reply:
> We could also enable crown unconditionally in CI if you would prefer (by checking if the CI environment variable is set).

`--use-crown` is good enough.

Reply:
Just to clarify - I only added `--use-crown` to the Linux CI tests, since the lints should be entirely platform independent (based on my limited understanding of crown). Are you okay with that?

Reply:
ok - done